src/_includes/section.edde.njk

```diff
@@ -1,40 +1,39 @@
 {% for item in collections.sections %}
   {% if item.data.type === 'edde' %}
-    <div id="edde" class="bg-edde-pattern xl:p-40 md:p-16 p-8 py-32">
-      <div class="flex flex-col text-center pb-4 xl:pb-16">
-        <h2 class="xs:text-md tracking-widest text-purple-650 mt-20">{{ item.data.title }}</h2>
+    <div id="edde" class="bg-edde-pattern py-16 px-8">
+      {# Title, Subtitle, First P #}
+      <div class="grid grid-cols-1 gap-2 text-center lg:px-36">
+        <h2 class="tracking-widest text-purple-650 mt-20">{{ item.data.title }}</h2>
         <div class="flex justify-center items-center flex-row flex-wrap">
-          <p class="xs:text-2xl sm:text-4xl font-bold p-2">
+          <p class="text-2xl font-medium tracking-wider p-2 sm:text-3xl">
             {{item.data.subtitle | safe}}
           </p>
-          <img class="xs:max-h-10 sm:max-h-16" src="/img/edde.svg" alt="EDDE Logo">
+          <img class="max-h-10 sm:max-h-16" src="/img/edde.svg" alt="EDDE Logo">
         </div>
-        <div class="text-1 p-4 sm:text-xl sm:px-14 md:px-2">
-          {{ item.templateContent | safe }}
-        </div>
-      </div>
-
-      <div class="grid grid-cols-1 xl:grid xl:grid-cols-3 md:gap-1 md:py-5 gap-y-3">
+        <p>{{ item.templateContent | safe }}</p>
+      </div> 
+      {# Loop through first row #}
+      <div class="grid grid-cols-1 gap-y-3 md:gap-1 md:py-5 xl:grid xl:grid-cols-3 xl:px-36">
```

Comment:
"grid grid-cols-1 xl:grid-cols-3 gap-y-3 md:gap-1 xl:px-36"

remove xl:grid -- redundant since we have it for mobile